emmo/utils.py

```diff
@@ -6,10 +6,20 @@
 import xml.etree.ElementTree as ET
 
 from rdflib import Graph, URIRef
+from rdflib.util import guess_format
 
 import owlready2
 
 
+# Format mappings: file extension -> rdflib format name
+FMAP = {
+    'n3': 'ntriples',
+    'nt': 'ntriples',
+    'ttl': 'turtle',
+    'rdfxml': 'xml',
```

Comment:
Should we not also add .owl as possible file extension?

Reply:
rdflib already recognises the .owl extension as the xml format, so it is not needed